doc/onboarding-extras.md

```diff
@@ -27,6 +27,7 @@
 | `test/*` | @nodejs/testing |
 | `tools/eslint`, `.eslintrc` | @not-an-aardvark, @silverwind, @trott |
 | async_hooks | @nodejs/diagnostics |
+| performance | @nodejs/performance |
```

Comment:
I considered it, but I'm not sure. That's probably a question for @nodejs/benchmarking.